3_calibrate/src/calibration_utils.R

```diff
@@ -50,36 +47,63 @@ run_glm_cal <- function(nml_obj,
   # get starting values for params that will be modified
   cal_starts <- sapply(cal_params, FUN = function(x) glmtools::get_nml_value(nml_obj, arg_name = x))
 
-  # define parscale for each cal param
-  # have to do all of this to match the WRR method of parscale Kw being a function of Kw:
-  parscale <- sapply(names(cal_parscale), FUN = function(x) {
-    if (class(cal_parscale[[x]]) == 'call') {
-      eval(cal_parscale[[x]], envir = setNames(data.frame(cal_starts[[x]]), x))
-    } else cal_parscale[[x]]
-  })
+  # # build obs data file path
+  # tmp_cal <- file.path('1_prep/out', cal_data_fl)
+
+  if(optimize == T) {
```

Comment:
Would suggest you edit to 
```r
if(optimize) { ...}
```

So `optimize == TRUE` is instead `optimize` and `optimize == FALSE` would be `!optimize`

Reply:
Done, but `optimize` was swapped for `calibrate`